scalableminds/webknossos
# Show clicked Bounding Box in  BB-Tab

### URL of deployed dev instance (used for testing):
- https://allowhavingselectedboundingbox.webknossos.xyz

### Steps to test:
- Create an annotation of your favourite dataset.
- Use the bbox tool to create a lot of bounding boxes.
- Whenever a new bounding box is created, the bounding box tab should scroll to the bottom and highlight this bounding box.
- Create a lot of bounding boxes to get into a scrolling state in the bbox tab
- When a bounding box is clicked while the bbox tool is active, the bbox should also be highlighted.
- Same goes for resizing

### TODOs:
- [x] Scroll selected bbox into view. Maybe https://github.com/ant-design/ant-design/issues/6570 helps -> :-1: 

### Open Questions
- [x] Should the context menu support to highlight a bounding box? -> Not necessary now.

### Issues:
- fixes [#7606](https://github.com/scalableminds/webknossos/issues/7606)

------
(Please delete unneeded items, merge only when none are left open)
